assets/src/apps/delivery/components/ActivityRenderer.tsx

```diff
@@ -353,6 +353,7 @@ const ActivityRenderer: React.FC<ActivityRendererProps> = ({
       }
       if (initObject.operator === 'bind to') {
         initStateBindToFacts[initObject.target] = snapshot[key] || '';
+        return acc;
```

Comment:
We don't send the BindTo trap states. I missed it with the new approach to fix the SW